> Unfortunately, you didn't keep a complete record of your work in this PR. No intermediate commits, no finished CI runs with failure logs. My e-mail notifications indicate that the port should support more platforms, but CI environment is too old. In that case the manifest shouldn't be restrictive. CI limitations need to go to the `scripts/ci.*` files.

I see, what would be good approach at this case?
I think we can enable linux for sure, as it represents 11-st GCC not 15-th during compilation.

As for Mac OS X and Android case it does not seem support this yet, since it probably represent very new Mac OS X buildtools 15.2 AppleClang 16.0.0.16000026 and very new android NDK r27c ANDROID-28 IMHO. Both have same issue during build. Alas clang-16 Mac OS X/clang-18 NDK are seem to be not supported yet.

```
Change Dir: '/Users/vcpkg/Data/b/orange-math/x64-osx-dbg'

Run Build Command(s): /usr/local/bin/ninja -v -v -j13 install
[0/2] /usr/local/bin/cmake -P /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/VerifyGlobs.cmake
[1/4] /Library/Developer/CommandLineTools/usr/bin/c++ -DOMATH_IMGUI_INTEGRATION -DOMATH_SUPRESS_SAFETY_CHECKS -DOMATH_USE_AVX2 -I/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include -isystem /Users/vcpkg/Data/installed/x64-osx/include -fPIC -g -std=gnu++2b -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX15.2.sdk -MD -MT CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o -MF CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o.d -o CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o -c /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx
FAILED: CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o 
/Library/Developer/CommandLineTools/usr/bin/c++ -DOMATH_IMGUI_INTEGRATION -DOMATH_SUPRESS_SAFETY_CHECKS -DOMATH_USE_AVX2 -I/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include -isystem /Users/vcpkg/Data/installed/x64-osx/include -fPIC -g -std=gnu++2b -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX15.2.sdk -MD -MT CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o -MF CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o.d -o CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx.o -c /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx
In file included from /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/3d_primitives/box.cpp:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/3d_primitives/box.hpp:7:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/triangle.hpp:5:
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/vector3.hpp:219:50: error: sorry, non-type template argument of type 'float' is not yet supported
  219 |         [[nodiscard]] std::expected<Angle<float, 0.f, 180.f, AngleFlags::Clamped>, Vector3Error>
      |                                                  ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/vector3.hpp:225:24: error: no viable conversion from returned value of type 'std::unexpected<Vector3Error>' (aka 'unexpected<omath::Vector3Error>') to function return type 'int'
  225 |                 return std::unexpected(Vector3Error::IMPOSSIBLE_BETWEEN_ANGLE);
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/vector3.hpp:227:33: error: sorry, non-type template argument of type 'float' is not yet supported
  227 |             return Angle<float, 0.f, 180.f, AngleFlags::Clamped>::from_radians(std::acos(dot(other) / bottom));
      |                                 ^
In file included from /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/3d_primitives/box.cpp:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/3d_primitives/box.hpp:7:
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/triangle.hpp:55:34: error: no return statement in constexpr function
   55 |         constexpr Vector3<float> side_a_vector() const
      |                                  ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/triangle.hpp:61:25: error: no return statement in constexpr function
   61 |         constexpr float hypot() const
      |                         ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/triangle.hpp:75:34: error: no return statement in constexpr function
   75 |         constexpr Vector3<float> side_b_vector() const
      |                                  ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/triangle.hpp:80:34: error: no return statement in constexpr function
   80 |         constexpr Vector3<float> mid_point() const
      |                                  ^
In file included from /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx:7:
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/collision/line_tracer.cpp:26:38: error: no member named 'side_a_vector' in 'omath::Triangle<omath::Vector3<float>>'
   26 |         const auto side_a = triangle.side_a_vector();
      |                             ~~~~~~~~ ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/collision/line_tracer.cpp:27:38: error: no member named 'side_b_vector' in 'omath::Triangle<omath::Vector3<float>>'
   27 |         const auto side_b = triangle.side_b_vector();
      |                             ~~~~~~~~ ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/collision/line_tracer.cpp:38:45: error: no member named 'm_vertex2' in 'omath::Triangle<omath::Vector3<float>>'
   38 |         const auto t = ray.start - triangle.m_vertex2;
      |                                    ~~~~~~~~ ^
In file included from /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/3d_primitives/box.cpp:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/3d_primitives/box.hpp:7:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/triangle.hpp:5:
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/vector3.hpp:121:20: error: cannot initialize object parameter of type 'const omath::Vector2<float>' with an expression of type 'const omath::Vector3<float>'
  121 |             return Vector2<Type>::dot(other) + z * other.z;
      |                    ^~~~~~~
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/collision/line_tracer.cpp:46:32: note: in instantiation of member function 'omath::Vector3<float>::dot' requested here
   46 |         const auto v = ray_dir.dot(q) * inv_det;
      |                                ^
In file included from /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx:10:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/engines/iw_engine/camera.cpp:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/camera.hpp:6:
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:13:30: error: constexpr variable 'k_abs_up' must be initialized by a constant expression
   13 |     constexpr Vector3<float> k_abs_up = {0, 0, 1};
      |                              ^          ~~~~~~~~~
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:14:30: error: constexpr variable 'k_abs_right' must be initialized by a constant expression
   14 |     constexpr Vector3<float> k_abs_right = {0, -1, 0};
      |                              ^             ~~~~~~~~~~
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:15:30: error: constexpr variable 'k_abs_forward' must be initialized by a constant expression
   15 |     constexpr Vector3<float> k_abs_forward = {1, 0, 0};
      |                              ^               ~~~~~~~~~
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:20:37: error: sorry, non-type template argument of type 'float' is not yet supported
   20 |     using PitchAngle = Angle<float, -89.f, 89.f, AngleFlags::Clamped>;
      |                                     ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:21:35: error: sorry, non-type template argument of type 'float' is not yet supported
   21 |     using YawAngle = Angle<float, -180.f, 180.f, AngleFlags::Normalized>;
      |                                   ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:22:36: error: sorry, non-type template argument of type 'float' is not yet supported
   22 |     using RollAngle = Angle<float, -180.f, 180.f, AngleFlags::Normalized>;
      |                                    ^
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/constants.hpp:24:42: error: use of undeclared identifier 'PitchAngle'
   24 |     using ViewAngles = omath::ViewAngles<PitchAngle, YawAngle, RollAngle>;
      |                                          ^
In file included from /Users/vcpkg/Data/b/orange-math/x64-osx-dbg/CMakeFiles/omath.dir/Unity/unity_0_cxx.cxx:10:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/source/engines/iw_engine/camera.cpp:4:
In file included from /Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/engines/iw_engine/camera.hpp:7:
/Users/vcpkg/Data/b/orange-math/src/v3.0.3-e6dfb25fc4.clean/include/omath/projection/camera.hpp:27:38: error: sorry, non-type template argument of type 'float' is not yet supported
   27 |     using FieldOfView = Angle<float, 0.f, 180.f, AngleFlags::Clamped>;
      |                                      ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/orange-math/portfile.cmake
+++ b/ports/orange-math/portfile.cmake
@@ -10,11 +10,6 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
-# We just let it built further as compiler would emit error since it wont recognize -mavx2 -mfma
-if(VCPKG_TARGET_IS_ANDROID)
-    vcpkg_replace_string("${SOURCE_PATH}/CMakeLists.txt" [[if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")]] [[if (0)]])
-endif()
-
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" OMATH_SHARED)
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/orange-math/portfile.cmake
+++ b/ports/orange-math/portfile.cmake
@@ -10,6 +10,11 @@ vcpkg_from_github(
     HEAD_REF master
 )
 
+# We just let it built further as compiler would emit error since it wont recognize -mavx2 -mfma
+if(VCPKG_TARGET_IS_ANDROID)
+    vcpkg_replace_string("${SOURCE_PATH}/CMakeLists.txt" [[if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")]] [[if (0)]])
+endif()
+
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" OMATH_SHARED)
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/orange-math/vcpkg.json
+++ b/ports/orange-math/vcpkg.json
@@ -4,6 +4,7 @@
   "description": "General purpose math library",
   "homepage": "https://github.com/orange-cpp/omath",
   "license": "MIT",
+  "supports": "windows | linux",
   "dependencies": [
     {
       "name": "vcpkg-cmake",
@@ -15,6 +16,7 @@
     }
   ],
   "default-features": [
+    "avx2",
     "imgui"
   ],
   "features": {

--- a/ports/orange-math/vcpkg.json
+++ b/ports/orange-math/vcpkg.json
@@ -4,7 +4,6 @@
   "description": "General purpose math library",
   "homepage": "https://github.com/orange-cpp/omath",
   "license": "MIT",
-  "supports": "windows",
   "dependencies": [
     {
       "name": "vcpkg-cmake",
@@ -16,13 +15,11 @@
     }
   ],
   "default-features": [
-    "avx2",
     "imgui"
   ],
   "features": {
     "avx2": {
-      "description": "Omath will use AVX2 to boost performance",
-      "supports": "!arm"
+      "description": "Omath will use AVX2 to boost performance"
     },
     "imgui": {
       "description": "Omath will define method to convert omath types to imgui types",

--- a/ports/orange-math/vcpkg.json
+++ b/ports/orange-math/vcpkg.json
@@ -3,7 +3,7 @@
   "version": "3.0.3",
   "description": "General purpose math library",
   "homepage": "https://github.com/orange-cpp/omath",
-  "license": "MIT",
+  "license": "Zlib",
   "supports": "windows | linux",
   "dependencies": [
     {

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -776,6 +776,7 @@ optional-bare:x64-windows-static-md=skip
 optional-bare:x64-windows-static=skip
 optional-bare:x64-windows=skip
 optional-bare:x86-windows=skip
+orange-math:x64-linux=fail
 orc:arm-neon-android=fail
 orc:arm64-android=fail
 orc:x64-android=fail

--- a/scripts/ci.feature.baseline.txt
+++ b/scripts/ci.feature.baseline.txt
@@ -2172,6 +2172,7 @@ openmvg(windows & arm) = fail # nmmintrin.h(17): fatal error C1189: #error:  Thi
 openscap[python](!windows) = feature-fails # CI image lacks swig
 opentelemetry-cpp[otlp](uwp) = feature-fails # --grpc_out: protoc-gen-grpc: The system cannot find the file specified. See https://github.com/microsoft/vcpkg/issues/34847
 openvdb[ax] = feature-fails # ax requires llvm version <= 14. Work when using a baseline + overwrite
+orange-math(linux) = fail # requires gcc15 or later
 osgearth[tools](osx) = feature-fails # Undefined _NSSearchPathForDirectoriesInDomains
 pangolin[core,eigen,examples,ffmpeg,gui,jpeg,lz4,module,openexr,png,pybind11,realsense,test,tiff,tools,vars,video,zstd](!(arm & windows)) = combination-fails # see https://github.com/microsoft/vcpkg/issues/31304
 poco[core,mariadb,mysql] = options # You can not install mariadb and mysql at the same time

--- a/versions/o-/orange-math.json
+++ b/versions/o-/orange-math.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "55aa9346aef1fa4d935e59167bb371aa6091cfb7",
+      "git-tree": "237d96964ad087cd579f6d21bc04181da48d1f2f",
       "version": "3.0.3",
       "port-version": 0
     },

--- a/versions/o-/orange-math.json
+++ b/versions/o-/orange-math.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "5be058afc332e9ed9ac5f58dac49729fd4c1a596",
+      "git-tree": "ae4b7b00db370ec2712fa46cc58a2147bc2f6cdc",
       "version": "3.0.3",
       "port-version": 0
     },

--- a/versions/o-/orange-math.json
+++ b/versions/o-/orange-math.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "ae4b7b00db370ec2712fa46cc58a2147bc2f6cdc",
+      "git-tree": "55aa9346aef1fa4d935e59167bb371aa6091cfb7",
       "version": "3.0.3",
       "port-version": 0
     },

